pkg/cvo/upgradeable.go

```diff
@@ -29,17 +29,18 @@ import (
 const (
 	adminAckGateFmt             string = "^ack-[4-5][.]([0-9]{1,})-[^-]"
 	upgradeableAdminAckRequired        = configv1.ClusterStatusConditionType("UpgradeableAdminAckRequired")
+	checkThrottlePeriod                = time.Minute
```

Comment:
nit: some day I'd like to break `pkg/cvo` up into more, smaller sub-packages.  But at the moment it's a big bucket doing lots of things, and we might want to use `checkUpgradeableThrottlePeriod` for clearer scoping here.  And add godocs explaining its intended semantics (not strictly required for these internal variables, but still potentially helpful to avoid accidental developer misuse)?

Reply:
Good point. I'm still pondering making it a member to allow unit-testing, then this const would be called `default...`.